src/racer/primitive.rs

```diff
@@ -4,8 +4,8 @@ use crate::nameres::{self, RUST_SRC_PATH};
 use rustc_ast::ast::{IntTy, LitIntType, UintTy};
 use std::path::PathBuf;
 
-const PRIM_DOC: &str = "libstd/primitive_docs.rs";
```

Comment:
Changing this to the new directory layout without also checking for the old layout will cause panics on any system that still has the old directory layout.